test/e2e/environment/environment.go

```diff
@@ -23,8 +23,12 @@ import (
 )
 
 var (
+	// Please keep the list in alphabetical order.
+
 	// The test does not work in UserNS (for example, `open /proc/sys/kernel/shm_rmid_forced: permission denied`).
 	NotInUserNS = framework.WithEnvironment(framework.ValidEnvironments.Add("NotInUserNS"))
+
+	// Please keep the list in alphabetical order.
```

Comment:
This comment at the bottom doesn't make a lot of sense here, but it is relevant for the larger lists: if it had been present earlier, then the diff in https://github.com/kubernetes/kubernetes/commit/6073d1cd3d58384d12a750bda749ff1922812be3#diff-337683b7f1560f35fe4a306d97241505a4b6e852d0aea485221d7f6ab67524d8 would have shown the request to keep the list sorted.

Adding that comment only at the top is not enough for that.